ROMFS/px4fmu_common/init.d/rc.io

```diff
@@ -3,6 +3,53 @@
 # PX4IO interface init script.
 #
 
+#
+# Check if PX4IO present and update firmware if needed.
+#
+if [ -f /etc/extras/px4io-v2.bin ]
+then
+	set IO_FILE /etc/extras/px4io-v2.bin
```

Comment:
Not now, but I'm wondering if we should push all of this logic directly into the px4io driver init.

For example this string is already hard coded in the common board config header. https://github.com/PX4/Firmware/blob/master/src/drivers/boards/common/board_common.h#L257